### ISSUE TITLE
Lyrics search

### ISSUE THREAD

**Issue by northross** [bradtraversy/vanillawebprojects#58]
429 Too Many Requests error.

It appears that there is a limit to how many requests that can be made to the lyrics page. Is there a way around this type of error? or is this simply how this API works?

I was able to fix the CORS error by adding the url to the beginning of the URL, is there something that can be added to fix the above problem?

**Comment by Broncs** [comment]
same here , did you find any solution ?

**Comment by northross** [comment]
No, I haven't gone back to check. Sorry. My buddy and I googled a bunch and we couldn't find anything.
